nui/src/main/java/org/terasology/nui/canvas/CanvasControl.java

```diff
@@ -27,14 +27,78 @@
 
     void postRender();
 
+    /**
+     * Call this method when the mouse position changes.
+     * This method does not support multiple pointers. Use {@link #processMousePosition(Vector2i, int)} for that case if supported.
```

Comment:
Maybe add a note here that calling this method will use the "first" or "main" pointer?